contracts/scripts/defender-actions/managePassThrough.js

```diff
@@ -0,0 +1,59 @@
+const { ethers } = require("ethers");
+const {
+  DefenderRelaySigner,
+  DefenderRelayProvider,
+} = require("@openzeppelin/defender-relay-client/lib/ethers");
+
+const passThroughAbi = require("../../abi/passThrough.json");
+const { logTxDetails } = require("../../utils/txLogger");
+
+// Entrypoint for the Defender Action
+const handler = async (event) => {
+  console.log(
+    `DEBUG env var in handler before being set: "${process.env.DEBUG}"`
+  );
+
+  // Initialize defender relayer provider and signer
+  const provider = new DefenderRelayProvider(event);
+  const signer = new DefenderRelaySigner(event, provider, { speed: "fastest" });
+
+  const OUSD = "0x2A8e1E676Ec238d8A992307B495b45B3fEAa5e86";
```

Comment:
It's ok to import the JS addresses. ie 
```js
const addresses = require("../../utils/addresses");
```

Reply:
Fixed in this commit: [a7705a7](https://github.com/OriginProtocol/origin-dollar/pull/2480/commits/a7705a7fff1ad9dd191effd7674d28dbdcd4f0e9)